frontend/src/pages/MetadataByMunicipality/MunicipalityMetadataResults.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import { NoResult } from './NoResult';
+import { SingleMetaDataResult } from './SingleMetaDataResult';
+
+const MunicipalityCategoriesContainer = styled.div`
+  height: 100%;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+`;
+const ResultsContainer = styled.div`
+  flex: 1;
+`;
+
+const MunicipalityMetadataResults = (props) => {
+  const { municipalityName } = props;
```

Comment:
I prefer having destructuring in the parameter body (props) when there's few like here